File: Hito1/CampeonatoDeFutbolPHP/Jugador.php
<?php
	class Jugador
	{
		private $nombreCompleto = "";
		private $apellidos = "";
		private $ci = "";
		private $edad = 0;

		function __construct($name, $lastName, $id, $age)
		{
			$this->nombreCompleto($name);
			$this->apellidos($lastName);
			$this->ci($id);
			$this->edad($age);
		}

		function getNombreCompleto()
		{
			return $nombreCompleto;
		}

		function setNombreCompleto($name){
			$this->nombreCompleto($name);
		}

		function getApellidos()
		{
			return $apellidos;
		}

		function setApellidos($lastName){
			$this->apellidos($lastName);
		}

		function getCi()
		{
			return $ci;
		}

		function setCi($id){
			$this->ci($id);
		}

		function getEdad()
		{
			return $edad;
		}

		function setEdad($age){
			$this->edad($age);
		}

		function imprimir(){
			echo "\t- ". $nombreCompleto;
			echo "\t- ". $apellidos; 
			echo "\t- ". $ci; 
			echo "\t- ". $edad; 
		}
	}
?>

File: Hito1/CampeonatoDeFutbol/src/main/java/danieldeveloper/campeonatodefutbol/Jugador.java
package danieldeveloper.campeonatodefutbol;

public class Jugador {
    private String nombreCompleto;
    private String apellidos;
    private String ci;
    private int edad;
    
    public Jugador(String name, String lastName, String ci, int age){
        this.nombreCompleto = name;
        this.apellidos = lastName;
        this.ci = ci;
        this.edad = age;
    }

    public String getNombreCompleto() { return nombreCompleto;}
    public void setNombreCompleto(String nombreCompleto) {this.nombreCompleto = nombreCompleto;}
    public String getApellidos() { return apellidos; }
    public void setApellidos(String apellidos) { this.apellidos = apellidos; }
    public String getCi() { return ci; }
    public void setCi(String ci) { this.ci = ci; }
    public int getEdad() { return edad; }
    public void setEdad(int edad) { this.edad = edad; }
    
    public void imprimir(){
        System.out.println("  Nombres: \t" + this.nombreCompleto);
        System.out.println("  Apellidos: \t" + this.apellidos);
        System.out.println("  C.I.: \t" + this.ci);
        System.out.println("  Edad: \t" + this.edad);
        System.out.println();
    }
}
